ci/matrix.xml

```diff
@@ -159,6 +159,7 @@
   <task id="bootstrap"><![CDATA[
     setJavaVersion $java
     npm install &&
+    sbt ++$scala tools/test &&
```

Comment:
I do not understand why the `tools` (the JVM part IIUC) move here.

Reply:
Because of https://github.com/scala-js/scala-js/blob/98d3b4e9e1182cef89fda312218cee4b212e3cba/project/Build.scala#L594-L597, which means `tools/test` depends on `library/packageBin` (for the same `scalaVersion`), and therefore we cannot do `tools/test` in 2.10 anymore.

Reply:
Fair enough :)